lib/measures/CreateTypicalBuilding/measure.rb

```diff
@@ -143,24 +203,34 @@ def run(model, runner, user_arguments)
     end
 
     @create = OpenstudioStandards::CreateTypical
-    runner.registerInfo("Begin typical model generation...")
+    runner.registerInfo('Begin typical model generation...')
 
     # If "Lookup From Model" is used, check that a climate zone was found in model. If not, return Error
     if climate_zone == 'Lookup From Model'
       climate_zone = standard.model_standards_climate_zone(model)
       if climate_zone == '' or climate_zone.nil?
-        error_message = "Error when looking up climate zone from model. Ensure the model has a valid ClimateZone"\
+        error_message = 'Error when looking up climate zone from model. Ensure the model has a valid ClimateZone'\
         " or ClimateZones objects with climate zone information present. \n**NOTE**: Geometry files typically do not"\
-        " have this information"
+        ' have this information'
         runner.registerError(error_message)
         return false
       end
 
     end
 
     # Fire off CreateTypical method
-    @create.create_typical_building_from_model(model, template, climate_zone: climate_zone,
-                                               hvac_system_type: hvac_type, user_hvac_mapping: hvac_mapping_hash)
+    @create.create_typical_building_from_model(model, template, climate_zone: climate_zone, add_hvac: add_hvac,
+                                               add_constructions: add_constructions,
+                                               wall_construction_type: wall_construction,
+                                               add_space_type_loads: add_space_type_loads,
+                                               add_daylighting_controls: add_daylighting,
+                                               add_swh: add_shw,
```

Comment:
It might be worth documenting how it's done.  Currently the `add_shw` parameter adds a service hot water system based on space type and sets parameters such as flow rates, temperatures, and overall system type based on that. The system types are based either "One Per Unit", "Shared", and "Dedicated".

It uses the `model_add_typical_swh` function:
https://github.com/NREL/openstudio-standards/blob/645b55538193ac865ce3d96537f8fd0be473d1ff/lib/openstudio-standards/prototypes/common/objects/Prototype.Model.swh.rb#L236